src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -15,7 +17,14 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 	}
 
 	@Override
-	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+	public void actionPerformed(ActionEvent e)
+	{
+		if (e.getActionCommand().contains("Figure Joe 1")) {
+			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		} else if (e.getActionCommand().contains("Figure Joe 2")) {
+			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		} else if (e.getActionCommand().contains("Figure Jane 1")) {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.